<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AsideClass</class>
 <widget class="QMainWindow" name="AsideClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aside</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="collectButton_">
        <property name="text">
         <string>collect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="collectionList_"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="nextButton_">
        <property name="text">
         <string>next action</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="nextActionList_"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="projectButton_">
        <property name="text">
         <string>project</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeView" name="projectTreeView">
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QPushButton" name="reviewButton_">
        <property name="text">
         <string>review</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCalendarWidget" name="calendarWidget"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1026</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="aside.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>collectButton_</sender>
   <signal>clicked()</signal>
   <receiver>AsideClass</receiver>
   <slot>collectThing()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>collectionList_</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>AsideClass</receiver>
   <slot>disposeCollection(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>413</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>433</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>projectButton_</sender>
   <signal>clicked()</signal>
   <receiver>AsideClass</receiver>
   <slot>addProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>575</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>602</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>collectThing()</slot>
  <slot>disposeCollection(QModelIndex)</slot>
  <slot>editProject()</slot>
  <slot>addProject()</slot>
 </slots>
</ui>
